uv.gyp

```diff
@@ -274,19 +274,44 @@
           },
         }],
         [ 'OS=="aix"', {
-          'sources': [ 'src/unix/aix.c' ],
+          'variables': {
+            'TrueOS': '<!(uname -s)',
+          },
+          'sources': [ 
+            'src/unix/aix-ibmi.c',
+          ],
           'defines': [
             '_ALL_SOURCE',
             '_XOPEN_SOURCE=500',
             '_LINUX_SOURCE_COMPAT',
             '_THREAD_SAFE',
-            'HAVE_SYS_AHAFS_EVPRODS_H',
           ],
-          'link_settings': {
-            'libraries': [
-              '-lperfstat',
-            ],
-          },
+          'conditions': [
+            [ '"<(TrueOS)"=="OS400"', {
+              'sources': [ 
+                'src/unix/ibmi.c',
+                'src/unix/posix-poll.c',
+                'src/unix/no-fsevents.c',
+                'src/unix/no-proctitle.c',
+              ],
+              'defines': [
+                '_PASE'
+              ],
+            }],
+            [ '"<(TrueOS)"!="OS400"', {
+              'sources': [ 
```

Comment:
Instead of checking for inequality with `OS400`, can't we do equality check with the expected name?

Reply:
That or fold it into the previous condition like this:
```python
[ '"<(TrueOS)"=="OS400"', {
  # ...
}, {
  # ...
}],
```

Reply:
The folded style is better. Updated the code.